src/msg/async/Event.cc

```diff
@@ -48,6 +52,10 @@ class C_handle_notify : public EventCallback {
  public:
   C_handle_notify(EventCenter *c, CephContext *cc): center(c), cct(cc) {}
   void do_request(uint64_t fd_or_id) override {
+#ifdef __linux__
+    eventfd_t value;
```

Comment:
nit, might want to always initialize local variables in order to silence warnings from static analyzer and might be helpful with debugging.

Reply:
Quite contrary, initializing a variable with an arbitrary useless value will hide real initializer bugs from sanitizers like Valgrind.
I mean, what value do you want to initialize it with?
If a static analyzer complains about this, then the static analyzer is broken.